### PR TITLE
chore: init metric

### PR DESCRIPTION
**Pull request by yogip07**
Closes # 

## ✅ Checks

- [v] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [v] All the tests have passed

### CODE REVIEW

File: telemetry/telemetry.go [hunk 1]
```diff
@@ -49,13 +61,38 @@ func (t *telemetryProvider) Run(ctx context.Context) (shutDownFuncs []ShutDownFu
...
+	}).WithResource(res)
+
+	metric, err = metric.WithGrpcExporter(ctx)
+	if err != nil {
+		return nil, err
+	}
+	metric, err = metric.WithHttpExporter(ctx)
+	if err != nil {
+		return nil, err
+	}
```
Comment (aldy505) [inline telemetry/telemetry.go]: Rasanya aneh ngeliat variable di override terus-terusan lewat function.

Comment (yogip07) [reply telemetry/telemetry.go]: terus gimana pak? soalnya perlu return error juga. Apa di-pass lewat param aja?

Comment (aldy505) [reply telemetry/telemetry.go]: yes, bisa dibikin options gitu.

```go
metric.New(metric.Config{ ... }, metric.WithResource(res), metric.WithGrpcExporter(ctx), metric.WithHttpExporter(ctx))

func New(config Config, options... Option) (*Metric, error) {}
```

Comment (yogip07) [reply telemetry/telemetry.go]: noted

Comment (yogip07) [reply telemetry/telemetry.go]: gw prefer gini 025898b tetep builder

**Review by aldy505 — APPROVED** [review]

**Comment by aldy505** [comment]
@YogiPristiawan ada conflict

**Comment by yogip07** [comment]
done

**Pull request merged**